lib/rspec/support/spec/library_wide_checks.rb

```diff
@@ -68,8 +68,13 @@ def load_all_files(files, preamble, postamble=nil)
       run_ruby_with_current_load_path(command, *options)
     end
 
-    # Ignore bundler warning.
-    stderr = stderr.split("\n").reject { |l| l =~ %r{bundler/source/rubygems} }.join("\n")
+    stderr = stderr.split("\n").reject do |l|
+      # Ignore bundler warning.
+      l =~ %r{bundler/(source/)?rubygems} ||
+      # Ignore bundler + rubygems warning.
+      l =~ %r{site_ruby/\d\.\d\.\d/rubygems} ||
+      l =~ %r{lib/bundler/rubygems}
```

Comment:
It looks like the `l =~ %r{bundler/(source/)?rubygems}` line will match anything this regex was match -- so do we really need it?